.github/workflows/daily-linkcheck.yml

```diff
@@ -0,0 +1,21 @@
+name: Daily link check
+on:
+  pull_request:
+  schedule:
+    - cron: '00 19 * * *' # every day at 19:00 UTC
+
+  workflow_dispatch:
+    # Manual trigger
```

Comment:
Please move the comment up one line, similar to line 5, and please leave 2 spaces before the #.